client/src/components/App/App.jsx

```diff
@@ -1,40 +1,109 @@
 import CssBaseline from '@material-ui/core/CssBaseline';
 import Header from '../Header/Header';
 import HeroBar from '../HeroBar/HeroBar';
 import ThemeProvider from '../ThemeProvider';
 import Form from '../Form/Form';
 import illustration from '../../../images/vaccine-illustration.jpg';
+import BackButton from '../BackButton/BackButton';
+import DownloadButton from '../DownloadButton/DownloadButton';
+import FactList from '../FactList/FactList';
 
 const App = props => (
   <ThemeProvider>
     <CssBaseline />
     <Header />
-    <HeroBar />
-    <Box
-      display="flex"
-      flexDirection="row"
-      justifyContent="center"
-    >
-      <Box mt={11} mr={4} maxWidth="530px">
-        <Form />
-      </Box>
-      <Box mt={2} ml={6} display="flex" flexDirection="column" alignItems="center">
-        <img src={illustration} alt="Woman receiving a vaccine" style={{maxWidth: '750px'}}/>
-        <Link
-          href="https://www.vecteezy.com/free-vector/epidemic-prevention"
-          color="secondary"
-          target="_blank"
-          rel="noopener"
-        >
-          Epidemic Prevention Vectors by Vecteezy
-        </Link>
-      </Box>
-    </Box>
 
+    <Router>    
+      <HeroBar />
+
+      <Switch>
+        <Route exact path="/">
+          <Box
+            display="flex"
+            flexDirection="row"
+            justifyContent="center"
+          >
+            <Box mt={11} mr={4} maxWidth="530px">
+              <Form />
+            </Box>
+            <Box mt={2} ml={6} display="flex" flexDirection="column" alignItems="center">
+              <img src={illustration} alt="Woman receiving a vaccine" style={{maxWidth: '750px'}}/>
+              <Link
+                href="https://www.vecteezy.com/free-vector/epidemic-prevention"
+                color="secondary"
+                target="_blank"
+                rel="noopener"
+              >
+                Epidemic Prevention Vectors by Vecteezy
+              </Link>
+            </Box>
+          </Box>
+        </Route>
+
+        <Route exact path="/data-found">
+          <BackButton />
+          <Box
+            display="flex"
+            flexDirection="row"
+            justifyContent="center"
+            mt={20}
+            mb={20}
+          >
+            SMART HEALTH CARD WILL GO HERE
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center">
+            <DownloadButton>
+              Download SMART&reg; Health Card
+            </DownloadButton>
+            <DownloadButton>
+              Download Printable PDF
+            </DownloadButton>
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center" my={3}>
+            <FactList />
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center">
+            <Box display="flex" flexDirection="column" maxWidth="750px">
+              <Typography variant="body2">
+                <p>
```

Comment:
getting one warning message of the following: 
![Screen Shot 2021-08-04 at 9 54 56 AM](https://user-images.githubusercontent.com/65190574/128203606-a4fa6181-0512-4169-a564-73a996c71bf1.png)

You can change the <p> in the <typography> element to <span> doing something like the following 

Not sure if <p> might have consequences later on with rendering and such

<Typography component={'span'} >

Reply:
fixed